php/class-coauthors-guest-authors.php

```diff
@@ -775,7 +775,15 @@ function manage_guest_author_filter_post_data( $post_data, $original_args ) {
 		if ( $user
 			&& is_user_member_of_blog( $user->ID, get_current_blog_id() )
 			&& $user->user_login != get_post_meta( $original_args['ID'], $this->get_post_meta_key( 'linked_account' ), true ) ) {
-			wp_die( esc_html__( 'Guest authors cannot be created with the same user_login value as a user. Try creating a profile from the user on the Manage Users listing instead.', 'co-authors-plus' ) );
+			// if user has selected to link account to matching user we don't have to bail
+			if ( $_POST['cap-linked_account'] > 0 && ( (int)$_POST['cap-linked_account'] === (int)$user->ID ) ) {
```

Comment:
We also will not need the `$_POST['cap-linked_account'] > 0` check since `$user->ID` will never equate to less than 0. For cases where we are not linking a user, we want to display an error 

```suggestion
			if ( (int)$_POST['cap-linked_account'] === (int)$user->ID ) {
```

Reply:
Resolving this feedback as it had been addressed in https://github.com/Automattic/Co-Authors-Plus/pull/748/commits/fb62271d24913f83a7a1d734a4e505213024c00e already.